Shared/ExampleMessages.cs

```diff
@@ -3,10 +3,10 @@ namespace LaunchDarkly.Xamarin.Example
 {
     public static class ExampleMessages
     {
-        public const string FlagValueIs = "The feature flag value is {0}";
+        public const string FlagValueIs = "Feature flag '{0}' is {1} for this user";
```

Comment:
I modeled this after the message in the hello-dotnet-client repo: https://github.com/launchdarkly/hello-dotnet-client/blob/main/Shared/DemoMessages.cs#L9. Let me know if this isn't right.

Reply:
Actually no, you can't do that, because the code in this app is different— it is only putting one variable value into the message. It's never safe to take a message that had a single variable placeholder like `{0}`, and change it to so that it instead has two placeholders `{0}` & `{1}` (or, as you've also done here, to change the _meaning_ of the existing `{0}` so that it is now a flag key instead of a value). The code and the message format need to agree on what they are trying to do.